File: src/main/java/com/shreyash/BankApplication/DTO/UserRequest.java
package com.shreyash.BankApplication.DTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class UserRequest {
    private String firstName;
    private String lastName;
    private String gender;
    private String email;
    private String address;
    private String phoneNumber;
    private String alternatePhoneNumber;

}

File: src/main/java/com/shreyash/BankApplication/dto/UserRequest.kt
package com.shreyash.BankApplication.dto;

import com.shreyash.BankApplication.authorities.AccountType
import lombok.*;
import java.math.BigDecimal

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
class UserRequest (
    @NonNull
    val panCardNumber:String,
    @NonNull
    val pinCode:String,
    @NonNull
    val accountType: AccountType,

    val creditMoney: BigDecimal

)
